src/components/Summary/PersonalDataFormStyles.ts

```diff
@@ -13,7 +17,8 @@ export const FormContainer = styled(Container)`
   gap: 1rem;
   margin-bottom: 2rem;
 
-  & .error-message {
-    color: red;
-  }
+  @media (max-width: 540px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme